### PR TITLE
WIP for new Branch linking domain

### PR DESCRIPTION
**Pull request by aeromusek**
*Please leave this PR open until the new Branch linking domain is live*

Some changes currently blocked until implementation details are known:

- [ ] **dynamic_link_builder.js** and **dynamic_link_builder_testing.js**: need to be rebuilt so they can handle both the new subdomains and legacy bnc.lt links
- [ ] **dashboard-guide.md** and **universal-app-links.md**: need to be updated with new custom subdomain procedure after that is established
- [ ] **universal-app-links.md**: need new `applinks` screenshots for Xcode and Xamarin Studio

### CODE REVIEW

**Comment by aeromusek** [comment]
@dmitrig01 is it be correct that we are now omitting the Branch key for dynamic long links, since it can now be inferred from the app-specific subdomain?

**Comment by dmitrig01** [comment]
@aeromusek yes. did you see the google doc?

**Comment by aeromusek** [comment]
@dmitrig01 got it! That'll be really nice and easy.

**Comment by aeromusek** [comment]
@dmitrig01 this is ready to go on my end (expecting more changes after launch, but that's how things always go.... 😁)

File: pages/getting-started/link-domain-subdomain.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+1. Check the **Enable custom domain** box.
+1. You should see a message telling you the status of your domain under the `Domain name` field. If you don't, please type your domain in again.
+1. Click the `Save` button.
+
+{% image src='/img/pages/getting-started/link-domain-subdomain/enable-custom-subdomain.png' 2-thirds center alt='successful custom subdomain configuration' %}
+
+### Custom ROOT domain (branch.com)
+
+1. Go to [Link Settings](https://dashboard.branch.io/#/settings/link){:target="_blank"} on the Branch dashboard, and find the **Custom Link Domain** section.
+1. Check the **Enable custom domain** box.{% image src='/img/pages/getting-started/link-domain-subdomain/enable-custom-domain.png' 2-thirds center alt='successful custom domain configuration' %}
```
Comment (dmitrig01) [inline pages/getting-started/link-domain-subdomain.md]: would you mind providing a different example, as we don't want people putting bnc in this box - how about branch.rocks?

File: pages/getting-started/link-domain-subdomain.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+### Custom SUBDOMAIN (go.branch.com)
+
+1. Create a CNAME for your subdomain and point it to `custom.bnc.lt`
+1. Go to [Link Settings](https://dashboard.branch.io/#/settings/link){:target="_blank"} on the Branch dashboard, and find the **Custom Link Domain** section.
+1. Check the **Enable custom domain** box.
+1. You should see a message telling you the status of your domain under the `Domain name` field. If you don't, please type your domain in again.
+1. Click the `Save` button.
+
+{% image src='/img/pages/getting-started/link-domain-subdomain/enable-custom-subdomain.png' 2-thirds center alt='successful custom subdomain configuration' %}
```
Comment (dmitrig01) [inline pages/getting-started/link-domain-subdomain.md]: similar here - can you provide the example myapp.branch.rocks or deeplink.branch.io or something – we don't want people putting custom.bnc.lt in

**Comment by aeromusek** [comment]
All set

**Pull request merged**